src/sonic-dhcp-server/tests/test_utils.py

```diff
@@ -26,6 +26,34 @@
         "expected_res": [["192.168.0.10", "192.168.0.10"]]
     }
 }
+validate_str_type_data = [
+    # type, value, expected_res
+    ["string", 123, False],
+    ["string", "123", True],
+    ["binary", "01020304ef", True],
+    ["binary", "01020304e", False],
```

Comment:
why this invalid?

Reply:
kea-dhcp-server only support octet-based binary, will add comments to explain